src/System.CommandLine/Properties/Resources.Designer.cs

```diff
@@ -0,0 +1,72 @@
+﻿//------------------------------------------------------------------------------
+// <auto-generated>
+//     This code was generated by a tool.
+//     Runtime Version:4.0.30319.42000
+//
+//     Changes to this file may cause incorrect behavior and will be lost if
+//     the code is regenerated.
+// </auto-generated>
+//------------------------------------------------------------------------------
+
+namespace System.CommandLine.Properties {
```

Comment:
❔ Is this repository using Arcade? If so, this file is not required.

Reply:
Can you elaborate on that? It does use Arcade, however Arcade is not required for developers. We have an environment variable that can disable many of the Arcade customizations on developer machines.

Reply:
Based on testing locally, there is stuff in the MSBuild pipeline that handles generating the needed code behind (similar to the what the .Designer.cs was doing with the embedded tool). Doesn't look like there is any additional overhead for developers so I just pushed the update.